fhir-search/src/test/java/com/ibm/fhir/search/uri/test/UriTest.java

```diff
@@ -62,6 +62,37 @@ public void testUriBadSecurity() throws URISyntaxException {
             incoming);
     }
 
+    @Test
+    public void testUriBadSecurity() throws URISyntaxException {
+        String incoming =
+                "https://localhost:9443/fhir-server/api/v4/_search?_count=10&_security=http://ibm.com/fhir/security&_fudge=tag&_page=1";
+        String requestUriString = incoming;
```

Comment:
maybe assign directly to requestUriString (however, it does model really well the existing code)

Reply:
i left it as 2 different variables just for consistency with the other test.  to me, its more clear this way that one is testing a truncated URL (with no query part) and the other is testing the incoming url in whole

Reply:
Yep, makes sense